R/render-plot.R

```diff
@@ -889,6 +889,15 @@ find_panel_info_non_api <- function(b, ggplot_format) {
   })
 }
 
+# Use public API for getting the unit's type (grid::unitType(), added in R 4.0)
+# https://github.com/wch/r-source/blob/f9b8a42/src/library/grid/R/unit.R#L179
+getUnitType <- function(u) {
+  if (getRversion() >= "4.0.0") {
```

Comment:
Safer to not use a version check, and check for the existence of `unitType` directly. Also, can use `getFromNamespace`.